cpp/src/gandiva/tests/decimal_test.cc

```diff
@@ -480,4 +480,107 @@ TEST_F(TestDecimal, TestCastFunctions) {
   EXPECT_ARROW_ARRAY_EQUALS(array_float64, outputs[4]);
 }
 
+TEST_F(TestDecimal, TestHashFunctions) {
```

Comment:
lets add a function that wraps hash functions too

like hash(col, hash(col2))

Reply:
done